README.md

```diff
@@ -42,6 +42,12 @@ This repo also contains code to help you build the entire .NET product end-to-en
 This is currently only supported on Linux.
```

Comment:
This sentence feels like something that better belongs in the support section.  Thoughts?